### PR TITLE
[semver:minor] Add support for container-secrets-updates

### PR DESCRIPTION
**Pull request by jychen7**
try to fix https://github.com/CircleCI-Public/aws-ecs-orb/issues/123

### CODE REVIEW

**Comment by jychen7** [comment]
I didn't change the terraform for integration test yet, since I am not sure whether it has the permission to write SSM Parameter Store :D

**Comment by gmemstr** [comment]
Having trouble validating this orb even though to my eye the YAML appears valid. Grabbing a second pair of eyes so we can get to the bottom of this.

File: src/commands/update-service.yml [hunk 1]
```diff
@@ -120,6 +134,7 @@ steps:
             family: << parameters.family >>
             container-image-name-updates: << parameters.container-image-name-updates >>
             container-env-var-updates: << parameters.container-env-var-updates >>
+            container-secrets-updates: << parameter.container-secrets-updates >>
```
Comment (KyleTryon) [inline src/commands/update-service.yml:137]: parameters*

File: src/jobs/deploy-service-update.yml [hunk 2]
```diff
@@ -172,6 +186,7 @@ steps:
       codedeploy-load-balanced-container-port: << parameters.codedeploy-load-balanced-container-port >>
       container-image-name-updates: << parameters.container-image-name-updates >>
       container-env-var-updates: << parameters.container-env-var-updates >>
+      container-secrets-updates: << parameter.container-secrets-updates >>
```
Comment (KyleTryon) [inline src/jobs/deploy-service-update.yml:189]: parameters*

File: src/jobs/update-task-definition.yml [hunk 3]
```diff
@@ -67,3 +81,4 @@ steps:
       family: << parameters.family >>
       container-image-name-updates: << parameters.container-image-name-updates >>
       container-env-var-updates: << parameters.container-env-var-updates >>
+      container-secrets-updates: << parameter.container-secrets-updates >>
```
Comment (KyleTryon) [inline src/jobs/update-task-definition.yml:84]: parameters*

**Review by KyleTryon — CHANGES_REQUESTED** [review]
This was harder to spot than I would have thought. There are three places where `parameter` is used and must be changed to `parameters`

File: src/commands/update-service.yml (hunk 1 above)
Comment (jychen7) [reply src/commands/update-service.yml]: sorry, my bad, thanks for spotting out

**Pull request closed without merging**